test/unit/utils/telemetry/collector_test.rb

```diff
@@ -0,0 +1,46 @@
+require 'inspec/utils/telemetry'
+require_relative '../../../helper.rb'
+
+class TestTelemetryCollector < Minitest::Test
+  def setup
+    @collector = Inspec::Telemetry::Collector.instance
+    @collector.reset
+  end
+
+  def test_collector_singleton
+    assert_equal Inspec::Telemetry::Collector.instance, @collector
+  end
+
+  def test_add_data_series
+    data_series = Inspec::Telemetry::DataSeries.new('/resource/File')
+    assert @collector.add_data_series(data_series)
+  end
+
+  def test_list_data_series
+    assert @collector.list_data_series
```

Comment:
I would expect tests on a listing method to:
 * verify it is empty at first
 * add a few data series
 * make sure you get those data series back
 * check the type of the list
 * check the type of the list elements

Reply:
Ok